tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -15952,25 +16014,29 @@ TEST_F(EnergyPlusFixture, Test_UnitarySystemModel_SubcoolReheatCoil)
     state->dataZoneEnergyDemand->ZoneSysEnergyDemand(1).RemainingOutputReqToHeatSP = -200.0;
     state->dataZoneEnergyDemand->ZoneSysMoistureDemand(1).RemainingOutputReqToDehumidSP = -200.0 / 2500940.0;
     // test COOLING condition
-    state->dataLoopNodes->Node(1).Temp = 24.0;      // 24C db
-    state->dataLoopNodes->Node(1).HumRat = 0.01522; // 17C wb
-    state->dataLoopNodes->Node(1).Enthalpy = Psychrometrics::PsyHFnTdbW(state->dataLoopNodes->Node(1).Temp, state->dataLoopNodes->Node(1).HumRat);
-    state->dataLoopNodes->Node(1).Press = state->dataEnvrn->OutBaroPress;
+    int zoneNodeNum = state->dataZoneEquip->ZoneEquipConfig(1).ZoneNode;
+    state->dataLoopNodes->Node(zoneNodeNum).Temp = 24.0;      // 24C db
+    state->dataLoopNodes->Node(zoneNodeNum).HumRat = 0.01522; // 17C wb
+    state->dataLoopNodes->Node(zoneNodeNum).Enthalpy =
+        Psychrometrics::PsyHFnTdbW(state->dataLoopNodes->Node(zoneNodeNum).Temp, state->dataLoopNodes->Node(zoneNodeNum).HumRat);
+    state->dataLoopNodes->Node(zoneNodeNum).Press = state->dataEnvrn->OutBaroPress;
```

Comment:
I changed this section to look up the zoneNodeNum, so in the future if these nodes are read in a different order, the unit test still works.